.github/workflows/test.yml

```diff
@@ -11,6 +11,8 @@ jobs:
       with:
         go-version: '1.19'
         check-latest: true
+    - name: Install expect
+      run: sudo DEBIAN_FRONTEND=noninteractive apt-get install -yq expect
```

Comment:
Is `expect` needed given that our use of `doppler setup` won't actually be interactive? (due to the token only having access to a single project/config)

Reply:
Now that you mention it, we can probably just grep for the output. Do we want to keep the pattern there for future cases where we might want to use it so we have a reference?

Reply:
Seems reasonable enough!